src/d3d8/d3d8_texture.h

```diff
@@ -103,7 +103,7 @@ namespace dxvk {
       return ptr;
     }
 
-    std::vector<Com<SubresourceType>> m_subresources;
+    std::vector<Com<SubresourceType, false>> m_subresources;

```

Comment:
See above about private Com. This one might need to stay private, I'm not sure.

Reply:
BloodRayne, BloodRayne2, Blowout, Railroad Tycoon 3, Firestarted and some of the D3D8 SDK apps seem to think otherwise. They do not behave correctly if we mess with public refcounts (especially on textures) beyond what is expected through d3d8 calls, and public COM objects also mess with them on assignment/release.